setup.py

```diff
@@ -30,6 +30,7 @@ def my_test_suite():
         "tqdm>=4.45.0",
         "networkx>=2.4",
         "requests>=2.23.0",
+        "scikit-network>=0.32.1",
```

Comment:
This should be an optional dependency if possible

Reply:
Moved the scikit-networks dependency to the optional section